src/docdb/nodes.ts

```diff
@@ -166,7 +168,7 @@ export class DocDBDocumentNode implements IDocumentNode {
 		const _self: string = this.data._self;
 		this._data = await new Promise<IDocDBDocumentSpec>((resolve, reject) => {
 			client.replaceDocument(_self, newData,
-				{ accessCondition: { type: 'IfMatch', condition: newData._etag } },
+				{ accessCondition: { type: 'IfMatch', condition: newData._etag }, partitionKey: this.partitionKeyValue || Object() },
```

Comment:
I realize that every time I access this.partitionKeyValue I'm also doing a falsey-check. Not placing the ( || Object() ) in the method since I don't know if _every_ API call needs an empty Object if partitionKeyValue is undefined/null